.appveyor.yml

```diff
@@ -0,0 +1,35 @@
+environment:
+  matrix:
+    - nodejs_version: "6"
+      configuration: publish
+    - nodejs_version: "7"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+
+before_build:
+  # Output useful info for debugging.
+  - node --version
+  - npm --version
+
+build_script:
+  - ./build.cmd
+
+cache:
+  - node_modules -> package.json
+  - '%APPDATA%\npm-cache'
+
+test: off
+
+deploy_script:
+  - ps: |
+      if ($ENV:CONFIGURATION -eq "publish")
+      {
+        "//registry.npmjs.org/:_authToken=`$`{NPM_TOKEN`}" | Out-File (Join-Path $ENV:APPVEYOR_BUILD_FOLDER ".npmrc") -Encoding UTF8
+        iex "npm config list --global"
+        iex "npm config list"
```

Comment:
We could remove the `iex "npm config list --global"` and `iex "npm config list"`, unless you want to leave them.